lib/src/models/language.dart

```diff
@@ -9,9 +9,9 @@ part 'language.g.dart';
 class Language {
   /// {@macro languages_struct}
   const Language({
-    required this.name,
-    required this.iso639,
-    required this.stationCount,
+    this.name = '',
+    this.iso639 = '',
+    this.stationCount = 0,
```

Comment:
what's the reasoning behind this change?

Reply:
the code crashed when iso639 was null.  It seems common.  I think it's always safest to do this non required = '' when reading data that could be null.